_posts/2020-10-02-serverless-kubernetes-cloudrun.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: Serverless Kubernetes with Cloudrun
+date: 2020-09-28
+tags: containers serverless docker kubernetes
+author: JK Gunnink
+image: img/cloudrun_header.png
+---
+
+![Cloudrun_Header]({{ site.base_url }}/img/cloudrun_header.png)
+
+A short while ago, Mr Brown-shoe himself [presented a talk at Yow! in Perth][yow-link] on the great debate of Kubernetes
+vs serverless, and which was to be the clear winner. Long story short it turned out that serverless is, by far a clear
+winner in nearly all categories of:
+
+- Security
+- Cost
+- Performance
+- Learning Curve
+- Development Experience
+- Operations
+
+If memory serves, the only category which Kubernetes came out a winner for according to the talk was in the performance
```

Comment:
wasn't there an inflection point on usage which made it more attractive to use Kube? ie. 10000 hits / min.. or something.. can we get his graph?

Reply:
I shall ask

Reply:
The graph was on compute cost specifically, and didn't have specific numbers attached, I can do a quick calc today and rig up another one quickly, k8s has got more expensive since then :P

Reply:
The performance aspect was yes k8s will be faster but it doesn't matter in 99.9% of cases
Also the DDD recording died somehow, something something Readify bias, I never managed to get ahold of it I redid it for Cloud Native London here: https://youtu.be/YzsKp6Je8eY?t=2301

Reply:
Resolved here: 0f0d7bf